core/src/main/java/com/exadel/aem/toolkit/bundle/lists/servlets/ListsServlet.java

```diff
@@ -48,25 +47,23 @@
 import com.adobe.granite.ui.components.ds.DataSource;
 import com.adobe.granite.ui.components.ds.EmptyDataSource;
 
+import com.exadel.aem.toolkit.bundle.CoreConstants;
+
 /**
  * Provides the collection of AEM resources that either represent AAT Lists or serve as folders for AAT lists
  * to be displayed in the AAT Lists console
  */
 @Component(
     service = Servlet.class,
     property = {
-        "sling.servlet.resourceTypes=/apps/authoring-toolkit/lists/datasources/list",
+        "sling.servlet.resourceTypes=/apps/authoring-toolkit/datasources/lists",
         "sling.servlet.methods=" + HttpConstants.METHOD_GET
     }
 )
-public class ChildResourcesDataSource extends SlingSafeMethodsServlet {
+public class ListsServlet extends SlingSafeMethodsServlet {
     private static final String LIST_TEMPLATE_NAME = "/conf/authoring-toolkit/settings/wcm/templates/list";
-    private static final String PN_PATH = "path";
-    private static final String PN_OFFSET = "offset";
-    private static final String PN_LIMIT = "limit";
-    private static final String REP_PREFIX = "rep:";
-    private static final String PN_RESOURCE_TYPE = "itemResourceType";
-    private static final String PATH_TO_JCR_CONTENT = String.format("/%s", JcrConstants.JCR_CONTENT);
+    private static final String NN_JCR_CONTENT = CoreConstants.SEPARATOR_SLASH + JcrConstants.JCR_CONTENT;
```

Comment:
not sure that 'NN' is suitable here, this is not a name, this is a relative path

Reply:
Renamed to `PATH...` (same to some other places)